web/js/map-OpenLayers.js

```diff
@@ -680,6 +680,12 @@ $.extend(fixmystreet.utils, {
             if (fixmystreet.map.popups.length) {
                 fixmystreet.map.removePopup(fixmystreet.map.popups[0]);
             }
+
+            // Ensures the permalink control is updated when the filters change
+            var permalink_controls = fixmystreet.map.getControlsByClass(/Permalink/);
```

Comment:
This only happens when the pins have loaded from the server, though - could you do this at the point the filters are changed/pin server request starts?

Reply:
👍